ludwig/api.py

```diff
@@ -197,10 +197,11 @@ def train(
         :param model_name: (str, default: `'run'`) name of the model that is
             being used.
         :param model_resume_path: (str, default: `None`) resumes training of
-            the model from the path specified. The difference with
-            model_load_path is that also training statistics like the current
-            epoch and the loss and performance so far are also resumed
-            effectively continuing a previously interrupted training process.
+            the model from the path specified. The model definition is restored.
+            In addition to model definition, training statistics and loss for
```

Comment:
and los and the state, would replace with a comma before the loss

Reply:
done